File: packages/internal/generators-files/spring-boot/application/shared/java/src/main/java/__packageDirectory__/__appClassName__.java
package <%= packageName %>;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication<% if(!minimal) { %>(scanBasePackages = "<%= basePackage %>")<% } %>
public class <%= appClassName %> {

  public static void main(String[] args) {
    SpringApplication.run(<%= appClassName %>.class, args);
  }

}

File: packages/internal/generators-files/spring-boot/application/shared/kotlin/src/main/kotlin/__packageDirectory__/__appClassName__.kt
package <%= packageName %>

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication<% if(!minimal) { %>(scanBasePackages = arrayOf("<%= basePackage %>"))<% } %>
class <%= appClassName %>

fun main(args: Array<String>) {
  runApplication<<%= appClassName %>>(*args)
}


